xml/System.Net.Http/DelegatingHandler.xml

```diff
@@ -23,9 +23,9 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This application normally instantiate this class and then set the inner handler or provide an inner handler in the constructor.  
+ An application should provide an inner handler either in the constructor or through the InnerHandler property before calling SendAsync() otherwise an InvalidOperationException will be thrown.
```

Comment:
InnerHandler property --> `<xref:System.Net.Http.DelegatingHandler.InnerHandler>` property 
SendAsync() otherwise an InvalidOperationException  --> 
`<xref:System.Net.Http.DelegatingHandler.SendAsync%2A>`; otherwise, an `<xref:System.InvalidOperationException>`

The `<xref> tags shouldn't be fenced in the text; I've fenced them only so markdown renders the tags.